bika/lims/exportimport/instruments/resultsimport.py

```diff
@@ -753,6 +753,44 @@ def _getZODBAnalysesFromReferenceAnalyses(self, objid, criteria):
 
         return analyses
 
+    def analysisHasCalcution(self, objid, analysis):
+        """ returns True or False"""
```

Comment:
Please write a more descriptive docstring, e.g.:

```python
def analysisHasCalcution(self, objid, analysis):
    """Checks if an Analysis has an calculation

    :param objid: AR ID or Worksheet's Reference Sample IDs
    :param analysis: Analysis Object
    :returns: True if one of the analyses has a calculation
    """
```

Btw, why do we need `analysis` in this method? It is not used at all.